docs/src/config.ts

```diff
@@ -79,6 +81,10 @@ export const SIDEBAR: Sidebar = {
       { text: 'Time display', link: 'en/media-time-display' },
       { text: 'Time range', link: 'en/media-time-range' },
       { text: 'Volume range', link: 'en/media-volume-range' },
+      { text: 'Clip selector (extra)', link: 'en/media-clip-selector' },
+    ],
+    "Learn More": [
+      { text: 'In the wild', link: 'en/in-the-wild' },
```

Comment:
I wonder if there's a better phrase to use here for navigational purposes. Spitballing...

- Who's using media-chrome?
- Public sightings
- Production demos
- Showcase
- Talks and examples

Reply:
I like "Showcase" 👍🏼 I think when we have more to show we can break these up into sub-pages but for now since we only have a few I think having a single "Showcase" page that has links to all of this stuff I think makes sense?